hack/make.sh

```diff
@@ -118,8 +118,10 @@ fi
 
 if [ -z "$DOCKER_CLIENTONLY" ]; then
 	DOCKER_BUILDTAGS+=" daemon"
-	if pkg-config libsystemd-journal 2> /dev/null ; then
+	if pkg-config 'libsystemd >= 209' 2> /dev/null ; then
```

Comment:
Ahh, I see what you mean -- could the Go file have something more complex like `libsystemd >= 209 || libsystemd-journal` ?